frontend/src/components/student/AssessmentExperience.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
```

Comment:
I had that thought, but then I feel like it makes sense to move all of its child components out of `student` as well. And then... that's like the whole `student` folder. LOL. 

I don't mind keeping it in `student` because at its core, I think the `AssessmentExperience` is basically just what the student sees, so I feel like its "student-y"? But don't feel super strongly about it.

But I'm open to ideas about renaming the folder or moving out the `AssessmentExperience` component and its children if you have suggestions on where to move it?

Reply:
makes sense, we can keep for now to avoid large refactor